plugins/org.jkiss.dbeaver.ui.editors.sql/plugin.xml

```diff
@@ -576,7 +576,10 @@
             <enabledWhen>
                 <with variable="activeEditor">
                     <adapt type="org.jkiss.dbeaver.ui.editors.sql.SQLEditor">
-                        <test property="org.jkiss.dbeaver.ui.editors.sql.canExplain"/>
+                        <and>
+                            <test property="org.jkiss.dbeaver.ui.editors.sql.canExecute" value="script"/>
```

Comment:
Why explain is enabled for script mode only?

Reply:
The SQL editor is the only "presentation" that can work with multiple statements at the same time. This check ensures the "Explain Plan" command is not available in VQB or AI Chat.
I'd say it's a hacky approach since it's based on assumptions.